### ISSUE TITLE
Fix UEV part costs

### ISSUE THREAD

**Issue by GTNH-Colen** [GTNewHorizons/GT-New-Horizons-Modpack#5514]
#### Which modpack version are you using?
2.0.8.3
#
UEV parts are simply absurdly expensive currently, we need to change the primary material used to infinity catalyst instead of pure infinity. A simple motor costs 32 infinity currently, given there are no endgame methods of generating more infinity this is just excessive. We could then perhaps use infinity in the creation of UIV parts (When we finally add those) and yes we do actually need those now given barts new circuits require a UIV circuit assembler, which currently cannot be crafted but does exist in NEI.

TLDR: Change the primary UEV material to infinity catalyst.

**Comment by 0lafe** [comment]
Iirc there is no tier past UEV for parts. Also endgame If production is a thing? 32 ingots is fine considering they're meant to be some expensive parts. Gen a few amps of UIV, and you can pump out ingots fairly quick

**Comment by DietmarKracht** [comment]
Endgame is not meant to be easy once you reach it.

**Comment by GTNH-Colen** [comment]
> Iirc there is no tier past UEV for parts

This is true and currently we don't really need UIV parts that badly, only real use I can see is so that barts new magneto UEV circuits can be produced as it requires a UIV circuit assembler. The primary point of this post however is just to discuss UEV costs.

> Gen a few amps of UIV

What planet are you living on? A few amps of UIV, say 3 for example is just over 100M eu/t? Even with MK3 fusion reactors creating say tin plasma (Which is arguably kinda OP) you'd still need around 20 MK3 fusion reactors and god knows how many turbines. Also you need to actually be able to pull these resources in somehow, a single motor under the current recipe uses 22k infinity catalyst while it's possible to produce a few attempting to produce UEV parts on any reasonable scale is totally out of the question with these costs. Maybe when we get a void miner option this will be more in line with reality but that currently just doesn't exist/work. Sure you could set up 4 ender quarries in the deep dark with absurdly large ranges but I think at that point we need to seriously consider the TPS/computational impact of that on users?

At the very least we need to create a new alloy with infinity and use that so that the total infinity used is diluted down. Same problem exists with the bio chain, it just uses far too much infinity.

> Endgame is not meant to be easy once you reach it.

Not sure why you of all people are disagreeing with me when you essentially said the same thing as me in another post? https://github.com/GTNewHorizons/NewHorizons/issues/4638

**Comment by Prometheus0000** [comment]
Is this still needed? It's easier to get infinity now, right?

**Comment by Prometheus0000** [comment]
@GTNH-Colen
